src/node_report.cc

```diff
@@ -189,6 +188,109 @@ unsigned int ProcessNodeReportVerboseSwitch(const char* args) {
   return 0;  // Default is verbose mode off
 }
 
+void SetVersionString(Isolate* isolate) {
+  Nan::TryCatch trycatch;
+  Nan::MaybeLocal<String> processProp = Nan::New<String>("process");
+  if (processProp.IsEmpty() || trycatch.HasCaught()) {
+    trycatch.Reset();
+    return;
+  }
+  Nan::MaybeLocal<Value> process = Nan::Get(
+    isolate->GetCurrentContext()->Global(),
+    processProp.ToLocalChecked());
+  if (process.IsEmpty() || process.ToLocalChecked()->IsUndefined()
+      || trycatch.HasCaught()) {
```

Comment:
I don't know how much core style is supposed to be followed but assuming it's the same style:

* the operator should go on the previous line
* line continuations in the Nan::Get() call should be indented with four spaces

The call to IsUndefined() looks like it should be `!process.ToLocalChecked()->IsObject()`?